checker/src/main/java/org/checkerframework/checker/index/qual/EnsuresLTLengthOf.java

```diff
@@ -78,4 +80,20 @@
     @JavaExpression
     @QualifierArgument("offset")
     String[] offset() default {};
+
+    /** An inner annotation which makes {@link EnsuresLTLengthOf} annotation repeatable */
+    @Documented
+    @Retention(RetentionPolicy.RUNTIME)
+    @Target({ElementType.METHOD, ElementType.CONSTRUCTOR})
+    @PostconditionAnnotation(qualifier = LTLengthOf.class)
+    @InheritedAnnotation
+    @interface List {
+        /**
+         * Programmers generally do not need to use this; it is created by Java when a programmer
+         * writes more than one {@link EnsuresLTLengthOf} annotation at the same location.
+         *
+         * @checker_framework.manual #java-expressions-as-arguments Syntax of Java expressions
```

Comment:
What is the point of this?  The annotation does not use Java expressions.

Reply:
Fixed this!